project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -38,7 +38,7 @@
      * the Servlet.
      */
     private static final LatLng LOCATION = new LatLng(32.080576, 34.780641); // Rabin Square TLV;
-    private static final ImmutableList<String> CUISINES = ImmutableList.of();
+    private static final ImmutableList<String> CUISINES = ImmutableList.of("sushi", "burger");
```

Comment:
How is this related to the PR?

Reply:
It isn't.. probably I changed it when I tested some things and didn't change back.